### ISSUE TITLE
UI for email watcher type accidentally removed?

### ISSUE THREAD

**Issue by chewi** [errbit/errbit#1077]
I understand the thinking behind #561 and don't have a problem with it except that the form fields for adding email-only watchers have been removed. The functionality seemingly still exists in the backend so perhaps this was accidental?

**Comment by stevecrozz** [comment]
We should probably remove the code that does email notifications for non-users since we don't support it any longer.

**Comment by chewi** [comment]
Perhaps it could be done as a Notification Service instead?

**Comment by stevecrozz** [comment]
@chewi That's exactly what I've had in mind, but I've been wanting to refactor the notification services the same way I refactored the issue trackers so they can be external plugins.

**Comment by adisos** [comment]
@stevecrozz 

So now there is no option to subscribe only one user? Either all users, or no one?

**Comment by chewi** [comment]
No, users can subscribe and unsubscribe themselves. This is about email notification with no user behind it.

**Comment by adisos** [comment]
It's strange, because I have no select box where I can choose watchers.

![image](https://cloud.githubusercontent.com/assets/592907/14630734/4fbb667e-0631-11e6-9ac3-66d4992d38f5.png)

**Comment by chewi** [comment]
Look at the application page (not the edit page) and you'll see a watch/unwatch button at the top right.

**Comment by adisos** [comment]
Oh, indeed. Thanks
